solutions/CSES Sliding Cost.mdx

```diff
@@ -0,0 +1,150 @@
+---
+id: cses-1077
+source: CSES
+title: Sliding Cost
+author: Isaac Noel
+---
+
+**Time Complexity:** $\mathcal{O}(N\log K)$
+
+## Abstract
+
+All elements in the window must be brought to the value of the median. Once we know the median, we may calculate the cost of the window by using a variety of range sums.
+
+
+<Warning title = "">
+
+		
+This problem requires an understanding of Sliding Median and Point Update Range Sums. Doing these first will make this problem significantly easier.
+
+[Point Update Range Sums module](https://usaco.guide/gold/PURS) [Sliding Median](https://usaco.guide/gold/sliding#problem-https://cses.fi/problemset/task/1076)
```

Comment:
Use `/gold/PURS` instead of `https://usaco.guide/gold/PURS` for an internal link